website/docs/user_guide/unleash_overview.md

```diff
@@ -22,7 +22,7 @@ Before you can connect your application to Unleash you need a Unleash server. Yo
 ### System Overview {#system-overview}
 
 ![A visual overview of an Unleash system as described in the following paragraph.](/img/unleash-architecture.svg 'System Overview')
-
+@Thomas - we need to review the text here, i updated the image (new: the orange text and one more option/setup)
```

Comment:
I did some stuff in 7f6794ae21f56d67407d8a33f540d15d7d17d7e0 which I think should cover it. Let me know if you disagree 😄